packages/lesswrong/components/notifications/NotificationsMenuButton.tsx

```diff
@@ -59,13 +97,96 @@ const NotificationsMenuButton = ({ unreadNotifications, open, toggle, className,
         {(unreadNotifications>0) ? <ForumIcon icon="Bell" /> : <ForumIcon icon="BellBorder" />}
       </IconButton>
     </Badge>
-  )
+  );
 }
 
-const NotificationsMenuButtonComponent = registerComponent('NotificationsMenuButton', NotificationsMenuButton, {
-  styles,
-  areEqual: "auto",
-});
+const hasKarmaChange = (
```

Comment:
I didn't notice any issues so prob good for now, and we can fix any bugs in the future. The only thing is that when I only have a new react then there is no change/indication on the bell icon. So I wonder if we should also show this karma star when you have a new react? Or maybe it's better to add the number of reacts to the number badge on the bell? I think by default there should be **some** indication on the bell, so maybe just go with the karma star unless @agnesstenlund says otherwise